src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public Rectangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    public double getFirstLeg() {
+        return this.firstLeg;
+    }
+
+    public void setFirstLeg(double firstLeg) {
+        this.firstLeg = firstLeg;
+    }
+
+    public double getSecondLeg() {
+        return this.secondLeg;
+    }
+
+    public void setSecondLeg(double secondLeg) {
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return this.firstLeg * this.secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: "
+                + getArea() + " sq. units, firstLeg : " + this.firstLeg + " units, "
+                + " secondLeg: " + this.secondLeg + " units, color: " + getColor());
+    }
```

Comment:
Use the `name()` method for getting the String representation of enum constants instead of `toString()` if you have an enum for colors.